packages/wrangler/src/cloudchamber/images/images.ts

```diff
@@ -68,6 +68,7 @@ export const registriesCommand = (yargs: CommonYargsArgvJSON) => {
 			(args) => configureImageRegistryOptionalYargs(args),
 			(args) =>
 				handleFailure(
+					`wrangler cloudchamber registries configure`,
```

Comment:
I'm pretty sure this string can be computed from the command config rather than duplicating it here.